src/components/BookForm/BookForm.js

```diff
@@ -1,13 +1,58 @@
-import React from 'react';
-
-const BookForm = () => (
-  <div>
-    <form>
-      <input type="text" name="title" placeholder="Book title" />
-      <input type="text" name="author" placeholder="Author" />
-    </form>
-    <button type="button">Add Book</button>
-  </div>
-);
+import React, { useState } from 'react';
+import { generate } from 'randomized-string';
+import { useDispatch } from 'react-redux';
+import { addNewBook, addBook } from '../../Redux/Books/Books';
 
+const BookForm = () => {
+  const [bookInit, setBook] = useState({
+    title: '',
+    category: '',
+    id: 0,
```

Comment:
- [ ] I kindly suggest that you make use of a unique id for each of your books. Kindly checkout uuid for implementation of unique id

### EXAMPLE
```
Command to install uuid:
npm install uuid

Importing uuid:
import { v4 as uuidv4 } from 'uuid';
uuidv4(); 

id:uuidv4() //Usage of  uuid
```

Reply:
Hello! @topeogunleye 

I really appreciate what you are doing I want just to tell you that if you chacked carefully I've used another generator called randomized-strings to get a unique id for each one of them.

Reply:
> Hello! @topeogunleye
> 
> I really appreciate what you are doing I want just to tell you that if you chacked carefully I've used another generator called randomized-strings to get a unique id for each one of them.

Great. Thank you. I'll remove the requirement.